packages/deploy/src/validator.ts

```diff
@@ -8,12 +10,16 @@ export interface Error {
   range?: [number, number, number];
 }
 
-export function parseAndValidate(input: string): {
+export async function parseAndValidate(
+  input: string,
+  specPath: string
```

Comment:
I think you should default `specPath` to `'.'` - that way you don't break unit tests which don't use it